src/Slider/Slider.css

```diff
@@ -15,39 +14,34 @@
 .slide-item{
   opacity : 0;
   transition: all 1s;
+  height: 500px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
 }
 .slide-item.active-slide{
   width: 100%;
   display: block;
+ 
   opacity: 1;
 }
-.slide-item img{
-  position: absolute;
-  top: 0;
-  left: 0;
-  bottom: 0;
-  right: 0;
-  width: 100%;
-  height: auto;
-}
-.slide-item.active-slide img{
-  position: static;
-  width: 100%;
-  height: auto;
-}
 .slider .labels{
   display: flex;
   position: absolute;
   bottom: 10px;
   left: calc(50% - 55px);
 }
 .slider .labels span{
-  width: 20px;
-  height: 5px;
+  width: 25px;
+  height: 7px;
+  border-radius: 2px;
   background-color: #fff;
   cursor: pointer;
   margin-left: 10px;
 }
+.slider .labels span:hover{
```

Comment:
will be better to avoid huge selector, you can use specific class for appropriate container instead.